File: node.js/consumer/consumer.js
import Fastify from 'fastify'

const fastify = Fastify({
  logger: true
})

fastify.post('/webhook', async (request, reply) => {
  try {
    const data = request.body
    
    console.log(`✅ Received message:`, data)

    return {
      status: 'success',
      received: data
    }
  } catch (error) {
    return reply.code(500).send({
      error: error instanceof Error ? error.message : 'Unknown error'
    })
  }
})

const start = async () => {
  try {
    await fastify.listen({ port: 6555, host: '0.0.0.0' })
  } catch (err) {
    fastify.log.error(err)
    process.exit(1)
  }
}

start() 

File: python/consumer/consumer.py
from flask import Flask, request, jsonify

app = Flask(__name__)

@app.route('/webhook', methods=['POST'])
def receive_message():
    try:
        data = request.json  
        
        print(f"✅ Received message: {data}")

        return jsonify({"status": "success", "received": data}), 200

    except Exception as e:
        return jsonify({"error": str(e)}), 500

if __name__ == '__main__':
    app.run(host='0.0.0.0', port=6555)